lib/openshift/cluster_logging.rb

```diff
@@ -0,0 +1,107 @@
+# represent a clusterlogging object
+module BushSlicer
+  class ClusterLogging < ClusterResource
+    RESOURCE = "clusterloggings"
+
+    ### represent status section ["collection", "curation", "logStore", "message", "visualization"]
+    private def collection_raw(user: nil, cached: true, quiet: false)
+      raw_resource(user: user, cached: cached, quiet: quiet).dig('status', 'collection')
+    end
+    private def curation_raw(user: nil, cached: true, quiet: false)
+      raw_resource(user: user, cached: cached, quiet: quiet).dig('status', 'curation')
+    end
+
+    private def log_store_raw(user: nil, cached: true, quiet: false)
+      raw_resource(user: user, cached: cached, quiet: quiet).dig('status', 'logStore')
+    end
+
+    private def message_raw(user: nil, cached: true, quiet: false)
+      raw_resource(user: user, cached: cached, quiet: quiet).dig('status', 'message')
+    end
+
+    private def visualization_raw(user: nil, cached: true, quiet: false)
+      raw_resource(user: user, cached: cached, quiet: quiet).dig('status', 'visualization')
+    end
+
+
+    private def fluentd_status_raw(user: nil, cached: true, quiet: false)
+      collection_raw(user: user, cached: cached, quiet: quiet).dig('logs', 'fluentdStatus')
+    end
+
+    private def curator_status_raw(user: nil, cached: true, quiet: false)
+      self.curation(user: user, cached: cached, quiet: quiet)['curatorStatus']
+    end
+
+    def fluentd_pods(user: nil, cached: true, quiet: false)
+      fluentd_status_raw(user: user, cached: cached, quiet: quiet)['pods']
+    end
+
+    def fluentd_ready_pods(user: nil, cached: true, quiet: false)
+      self.fluentd_pods(user: user, cached: cached, quiet: quiet)['ready']
+    end
+
+    def fluentd_failed_pods(user: nil, cached: true, quiet: false)
+      self.fluentd_pods(user: user, cached: cached, quiet: quiet)['failed']
+    end
+
+    def fluentd_notready_pods(user: nil, cached: true, quiet: false)
+      self.fluentd_pods(user: user, cached: cached, quiet: quiet)['notReady']
+    end
```

Comment:
All `fluentd*_pods` methods can be private as far as I can tell. If they are to be public, then they better return `Pod` objects instead of some unknown structure.